### ISSUE TITLE
Feature Request: option to associate to a place.

### ISSUE THREAD

**Issue by lordmcfuzz** [moom0o/PhotoSphereStudio#10]
Problem: Photos uploaded via this tool are not associated with any place and show up on the contributions history as  "Unknown Place". There does not seem to be an ability to edit and associate a place to an uploaded photo in google maps.

Request: Ability to select a place when uploading, So that both gps location and place information can be complete.

--------------------------

According to the API, uploading a photo can be associated with a Place. https://developers.google.com/streetview/publish/reference/rest/v1/photo#Place

It would be awesome to have the option to upload the photosphere with the association to the place as that would bridge the last gap of the functionality that we lost when google sunsetted their app.

I'm not sure how the Place Ids would be looked up. Personally if there was a manual workflow to look them up, I would not mind a simple text field for the photo POST. aka while something fancy would be cool but simple functionality would go a long way as long as the manual workflow is clear.

----------------------------

Thank you so much for developing and hosting this. I like recording parks that are not really visible from normal street view lines but also not large enough to warrant taking a video for a blue line.

FYI [map.winscloud.net](https://map.winscloud.net/) seems to be down.

**Comment by alexiscoutinho** [comment]
Yeah, this feature would be highly appreciated.

**Comment by milanomaster** [comment]
Yes. Any heads up on this? I'd also like to label my Panoramas, rather than them being 'Unknown Place' and lacking a custom title!

**Comment by WinsDominoes** [comment]
Hi everyone! I apologize for not seeing this feature request earlier, the main developer hasn't been active for a while and he is probably busy with his IRLs. 

I've read the Google Maps API Documentation and it is indeed possible to do so, I'll work on this and will provide an update soon. 

Thank you!

**Comment by VasyaPupkins** [comment]
Oh man. We need this so much.😭

**Comment by moom0o** [comment]
Has been added! Under advanced options 😃
